### ISSUE TITLE
Upload page bogs down after uploading many images

### ISSUE THREAD

**Issue by pnorman** [mapillary/mapillary_issues#198]
I uploaded a 500+ image sequence. After it had uploaded and was stuck at 99% the page was extremely slow.

Reloading fixed the slowness, and also allowed me to upload.

I haven't observed the problem on small sequences.

**Comment by ToeBee** [comment]
I see the browser (Chromium in my case) eating up one core worth of CPU power both during upload and after it has finished a large group of pictures. As soon as I refresh the page, the CPU fan on my laptop slows down or even completely shuts off.

**Comment by knikel** [comment]
We have addressed this issue in the latest iteration of the manual upload. We have also introduced an upload cap to encourage uploading photos is smaller batches (the cap can be changed to whatever amount you want by pressing **Manual upload settings** link). Unfortunately the browser performance will go drastically down when trying to upload a huge amount of photos at once, yet, it should be better to what you have experienced until now.

If it is important to upload a lot of images in one sequence, what we suggest to do is to upload eg. 100-400 images at once, when they are on the server, but not published yet, press the **Upload more** button, upload another 100-400 photos, and then press **Publish** when you're happy with your upload. It is not the best way to handle large uploads, but we're not the only ones having this issue (eg. Flick allows to upload max. 100 photos at once).
